test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,31 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // contains null
+        assertContainsNull(null, null);
```

Comment:
This is the last week LO and should not be inside this PR.
Do remember to create a clean branch every time you work on a new feature/modification.